input/_partials/_head.cshtml

```diff
@@ -10,10 +10,6 @@
   {
   <title>@Model.GetString("Title")</title>
   }
-  @if (string.Equals(Model.GetString("LayoutRef"), "blog", StringComparison.OrdinalIgnoreCase))
-  {
-    <meta http-equiv="refresh" content="5; url = /blog" />
```

Comment:
This is actually a bug, you don't want it - you'll notice on the current site where it's still included, every blog page redirects back to the "/blog" page after 5 seconds!

Reply:
yikes!